main.go

```diff
@@ -24,8 +24,9 @@ var osFileSystem = perfTestUtils.OsFS{}
 // Command line arguments:
 var configFilePath string
 var checkTestReadiness bool
-var verbose bool
-var debug bool
+var boolVerbose bool
+var boolDebug bool
+var configOverrides *perfTestUtils.Config
```

Comment:
Renamed Boolean options for readability.
The configOverrides container will be populated from defaults + command line settings, and will be used to override the configurationSettings container loaded from the config.xml file.